precompile/contracts/warp/predicate_test.go

```diff
@@ -156,8 +157,7 @@ func createWarpMessage(numKeys int) *avalancheWarp.Message {
 // and packs it into predicate encoding.
 func createPredicate(numKeys int) []byte {
```

Comment:
To align with the prior behavior (and the documentation on this function) it seems like we should be returning `predicate.New` from here.